packages/lib-classifier/test/stubPanoptesJs/stubPanoptesJs.js

```diff
@@ -1,14 +1,17 @@
 
+function createResponse (url, factories) {
+  // Assuming url is in the format '/resource_type/resource_id'
+  const endpoint = url.split('/')[1]
+  const response = {}
+  response[endpoint] = [factories[endpoint]]
+  return Promise.resolve({ body: response })
+}
+
 export default function stubPanoptesJs (factories) {
   return {
     panoptes: {
-      get: function (url, query) {
-        // Assuming url is in the format '/resource_type/resource_id'
-        const endpoint = url.split('/')[1]
-        const response = {}
-        response[endpoint] = [factories[endpoint]]
-        return Promise.resolve({ body: response })
-      }
+      get: (url) => createResponse(url, factories),
```

Comment:
Would it be worth wrapping these in spies, for convenience?

Reply:
Probably? I'd like to rethink what this helper function is doing separately though and build it out for mocking the other HTTP methods. And perhaps abstract it out to be a part of panoptes.js lib instead and available to other consumers.